hugo_site/content/hospitality.md

```diff
@@ -0,0 +1,79 @@
+# Hospitality
+
+It's a good idea to book your accommodation early on. One option is to buy a ticket and use the attendee channels to organize accommodation with others (this solution is coming up shortly). You may also prefer a classic hotel or hostel.
+
+It's a good idea to book your accommodation early on. One option is to buy a ticket and use the attendee channels to organize accommodation with others (this solution is coming up shortly). You may also prefer a classic hotel or hostel.
+
+It's a good idea to book your accommodation early on. One option is to buy a ticket and use the attendee channels to organize accommodation with others (this solution is coming up shortly). You may also prefer a classic hotel or hostel.
+
```

Comment:
Something went wrong here, I guess when they say that it's "Beta", they actually mean "Alpha" :tada: